cmd/format-disk-cache.go

```diff
@@ -172,6 +171,8 @@ func loadFormatCache(ctx context.Context, drives []string) ([]*formatCacheV2, bo
 			migrating = true
 		}
 		formats[i] = formatV2
+
+		f.Close()
```

Comment:
You are introducing a bug here by not closing the `fd` when the `formatMetaCacheV1()` returns an error. You are basically leaking fd now than before. 

Remember `defer` is stacked up here and this is a `load()` time call there is actually no real memory build up here. 

This happens during startup, and whenever the code finishes all the fd's are closed automatically.

Cached drives are usually like few, even if there are 1000 of them it is hardly anything for a modern system.

Reply:
You are right, I miss the `formatMetaCacheV1() ` error before